cmd/maya-apiserver/cstor-operator/cspc/operations.go

```diff
@@ -58,11 +59,11 @@ func (pc *PoolConfig) expandPool() error {
 
 		// Pool expansion for raid group types other than striped
 		if len(pool.RaidGroups) > len(cspObj.Spec.RaidGroups) {
-			cspObj = addGroupToPool(&pool, cspObj)
```

Comment:
this comment is for the code above this..
we shouldn't be returning errors if `nodeselect.GetNodeFromLabelSelector` fails or `getCSPWithNodeName` fails..
let us track the errors, and do a print at the end.. probably, by sending to caller or here itself